### PR TITLE
Add sorting to Issuer BadgeClasses

### PR DESCRIPTION
**Pull request by nqthqn**

### CODE REVIEW

File: apps/issuer/models.py [hunk 1]
```diff
@@ -629,6 +630,7 @@ def get_filtered_json(self, excluded_fields=('@context', 'id', 'type', 'name', '
     def cached_badgrapp(self):
         return self.cached_issuer.cached_badgrapp
 
+BadgeClass._meta.get_field('created_at').db_index = True
```
Comment (coffindragger) [inline apps/issuer/models.py]: we should not use the non-public _meta class

Comment (nqthqn) [reply apps/issuer/models.py]: 👍 Maybe we should add a param to `AllBadgeClassesList` for sorting.

Comment (nqthqn) [reply apps/issuer/models.py]: in `apps/issuer/api.py`

**Comment by nqthqn** [comment]
Just leaving a note here. If we index and order by created_at in BaseAuditedModel it will affect:

- ExternalTool
- BadgeInstance
- RecipientGroup
- Issuer
- BadgeClass
- ExternalToolUserActivation
- TermsAgreement
- TermsVersion
- BackpackCollection

**Comment by coffindragger** [comment]
db_index for created_at is being tracked on BGR-2786

**Pull request closed without merging**